ember-core/shared/src/main/scala/org/http4s/ember/core/ChunkedEncoding.scala

```diff
@@ -140,20 +142,37 @@ private[ember] object ChunkedEncoding {
     }
 
   private val lastChunk: Chunk[Byte] =
-    Chunk.byteVector((ByteVector('0') ++ crlf ++ crlf).compact)
+    Chunk.byteVector((ByteVector('0') ++ crlf).compact)
 
   /** Encodes chunk of bytes to http chunked encoding.
     */
-  def encode[F[_]]: Pipe[F, Byte, Byte] = {
+  def encode[F[_]: Functor](trailers: F[Headers]): Pipe[F, Byte, Byte] = {
```

Comment:
Just making sure we actually set this key, for every entity we are writing?

Reply:
https://github.com/http4s/http4s/blob/series/0.23/core/shared/src/main/scala/org/http4s/Message.scala#L210 - Resolving my own question